### PR TITLE
Added Hungarian(Magyar) translation

### PR DESCRIPTION
**Pull request by TheOneEyedGrimReaper**

### CODE REVIEW

File: auto h-encore/Language.cs [hunk 1]
```diff
@@ -967,6 +967,99 @@ public static class Language {
...
+                { "log_GetCMA", "CMA titkosító kulcs megszerzése AID al {0}" },
+                { "log_GotCMA", "CMA titkosító kulcs megszerezve {0}" },
+                { "log_Packaging", "h-encore csomagolása {0} psvimgtools al..." },
+                { "log_MoveToQCMA", "h-encore fájlok mozgatása a QCMA APP környvtárába...\r\n" },
+                { "log_Finished", "auto h-encore Végzett!!\r\n" },
+
+                { "title_Main", "auto h-encore" },
+                { "title_Import", "Létező fájlok importásása" },
+                { "title_Warning", "Figyelmeztetés" },
+                { "title_Error", "Hiba" }
```
Comment (barbabarros) [inline auto h-encore/Language.cs:1062]: } }

**Review by barbabarros — CHANGES_REQUESTED** [review]

**Comment by noahc3** [comment]
Thank you!

File: auto h-encore/Language.cs (hunk 1 above)
Comment (TheOneEyedGrimReaper) [reply auto h-encore/Language.cs:1062]: @noahc3  Thanks.
how soon may expect the builded version with the translation?

Comment (noahc3) [reply auto h-encore/Language.cs:1062]: I'm going to add 32bit support to h-encore then bundle it with 0.4.4. Soon.

**Comment by noahc3** [comment]
@TheOneEyedGrimReaper Sorry for the delay. v0.4.4 is now out with this translation.

**Pull request merged**